PhysicsTools/Heppy/python/analyzers/objects/PhotonAnalyzer.py

```diff
@@ -182,5 +278,7 @@ def process(self, event):
     etaMax = 2.5,
     gammaID = "PhotonCutBasedIDLoose",
     do_mc_match = True,
+    do_randomCone = False,
```

Comment:
@mangano @pandolf : in addition to adding the default here, you have to add it also to `susyCore_modules_cff.py`.
Can you add a commit for this and update the PR?